tests/CMakeLists.txt

```diff
@@ -4,3 +4,18 @@ TARGET_INCLUDE_DIRECTORIES(test_edge_sensor PRIVATE ${EDGE_REQUIRE_PKGS_INCLUDE_
 TARGET_LINK_LIBRARIES(test_edge_sensor -L${PROJECT_BINARY_DIR}/src edge-sensor)
 
 INSTALL (TARGETS test_edge_sensor DESTINATION ${BIN_INSTALL_DIR})
+
+# GoogleTest requires at least C++11
+SET(CMAKE_CXX_STANDARD 11)
```

Comment:
let's consider to set 14 as current nnstreamer requires it.
https://github.com/nnstreamer/nnstreamer/blob/50d82aa029c72053004cc35743ba54a79560bc0d/meson.build#L12

Reply:
Good! Updated.